README.md

```diff
@@ -19,32 +19,19 @@ With the [DTDL extension for Visual Studio Code](https://marketplace.visualstudi
 You could use the command palette to create interface from predefined or customized templates. 
 
 - In Visual Studio Code, select **View > Command Palette** to open the VS Code command palette.
-- In the command palette, enter and run the command **DTDL: Create Interface** 
-- Follow the instruction to assign the interface name and choose a template.  
-- A JSON file will be created in the current folder. The file name is based on the input interface name. 
-- You should replace the **{company}** field in @id with your own company’s name. Note that @id is the path component of the [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier-dtmi) (DTMI) and should follow the DTMI rule to uniquely identify the device model.
-- You could extend the interface with new types defined in [DTDL v2](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md). 
+- In the command palette, enter and run the command **DTDL: Create Interface**. 
+- Follow the instruction to assign the interface name. 
+- If there are multiple templates existing, choose a template you need. Otherwise, the extension will the basic template as default.
```

Comment:
Do we need to mention that the template should be a valid JSON file? and the file extension should be *.json

Reply:
Resolve this in the section which telling the customer how to store the templates.